bom/pom.xml

```diff
@@ -427,14 +427,34 @@ quarkus.vertx.max-event-loop-execute-time=${max.event-loop.execute-time:20000}
       </dependency>
       <dependency>
         <groupId>org.eclipse.hono</groupId>
-        <artifactId>hono-service-command-router</artifactId>
+        <artifactId>hono-service-command-router-base</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.eclipse.hono</groupId>
+        <artifactId>hono-service-command-router-infinispan</artifactId>
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.eclipse.hono</groupId>
+        <artifactId>hono-service-command-router-redis</artifactId>
+        <version>${project.version}</version>
+      </dependency>
```

Comment:
I created two "concrete" modules here that will produce docker images: `hono-service-command-router-infinispan` and `hono-service-command-router-redis`.

The problem with that is that we'd be changing the legacy name of the command router service (`hono-service-command-router`). So the question is whether I should instead use the legacy name for the Infinispan implementation, in order to maintain some sort of a backwards compatibility? It feels "cleaner" to me to explicitly name the docker images with the technology that they support and that's the way it's done in the device-registry service (one is explicitly named `-mongodb` and the other one is `-jdbc`). But that could potentially cause some trouble where the old image name suddenly stops being produced.

Another solution to this could be to have the pipeline that builds and publishes the images tag the `-infinispan` image additionally with the old name as well?